schema.graphql

```diff
@@ -3,7 +3,7 @@ type Colony @entity {
   ensName: String
   orbitAddress: String
   colonyChainId: BigInt!
-  token: String # Address
+  token: Token # Address
```

Comment:
This is a nitpick, but since this is an object now, the `# Address` comment is wrong :)

Reply:
I've gone back and forth on this. You still set the property to the address, as the ID of the token is the token address, and the type says "there's an object of this type with a corresponding ID". I can go either way :smile: